### PR TITLE
HJ-250 - Address PyPi deprecation warnings

### PR DESCRIPTION
**Pull request by andres-torres-marroquin**
Closes #HJ-250
### Description Of Changes

Address PyPi deprecation warnings on Pypi

### Code Changes

* Updated name on setup.py

### Steps to Confirm

1.  While being on this branch, `nox -s "tag(push)"` you should not receive an email about the deprecation warning.

### Pre-Merge Checklist

* [ ] Issue requirements met
* [ ] All CI pipelines succeeded
* [ ] `CHANGELOG.md` updated
* Followup issues:
  * [ ] Followup issues created (include link)
  * [ ] No followup issues
* Database migrations:
  * [ ] Ensure that your downrev is up to date with the latest revision on `main`
  * [ ] Ensure that your `downgrade()` migration is correct and works
    * [ ] If a downgrade migration is not possible for this change, please call this out in the PR description!
  * [ ] No migrations
* Documentation:
  * [ ] Documentation complete, [PR opened in fidesdocs](https://github.com/ethyca/fidesdocs/pulls)
  * [ ] Documentation [issue created in fidesdocs](https://github.com/ethyca/fidesdocs/issues/new/choose)
  * [ ] If there are any new client scopes created as part of the pull request, remember to update public-facing documentation that references our scope registry
  * [ ] No documentation updates required

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def optional_requirements(
 ## Package Setup ##
 ###################
 setup(
-    name="ethyca-fides",
+    name="ethyca_fides",
```
Comment (erosselli) [inline setup.py:57]: do we know if this change will affect anything in how fides is installed or imported?

Comment (adamsachs) [reply setup.py:57]: @andres-torres-marroquin one thing to test specifically may be that fidesplus can build with the package that's published from this branch

Comment (adamsachs) [reply setup.py:57]: @andres-torres-marroquin one thing to test specifically may be that fidesplus can build with the package that's published from this branch

**Review by adamsachs — APPROVED** [review]
@andres-torres-marroquin based on your testing, let's go ahead and get this merged (and the further it is away from the release cycle, the more time it gives us to iron out any kinks 👍 ).

before merging though, can you address the git history? this PR shouldn't be showing 26 commits...

**Pull request merged**